docs/helm-broker/03-03-create-addons-repo.md

```diff
@@ -110,9 +110,84 @@ spec:
     - url: "github.com/kyma-project/addons//addons/index.yaml"
     # bitbucket.org URL with no prefix. It is automatically interpreted as a Git repository source.
     - url: "bitbucket.org/kyma-project/addons//addons/index.yaml"
+    # Git SSH protocol with a reference to a secret containing base64 encoded SSH private key
+    - url: "git::ssh://git@github.com/kyma-project/private-addons.git//addons/index.yaml?sshkey={key}"
+      secretRef:
+        name: auth
+        namespace: stage
 ```
 
->**NOTE:** For now, the SSH protocol is not supported.
-
   </details>
 </div>
+
+## Authorization
+
+The AddonsConfiguration and ClusterAddonsConfiguration custom resources allows you to define authorization as a part of the URL. For more details please read the [go-getter protocols](https://github.com/hashicorp/go-getter/blob/master/README.md#general-all-protocols) description.
+ 
+The Helm Broker supports authorization using the following protocols:
+ 
+<div tabs>
+  <details>
+  <summary>
+  HTTP/HTTPS
+  </summary> 
+ 
+You can prepend a section `username:password@` to the hostname in the URL to define basic authentication crdentials, for example:
+```
+https://admin:secretPassword@repository.addons.com/index.yaml
+```
+
+You can create a Secret resource which contains credentials and reference it in the repository URL definition using templating:
+
+Create a Secret:
+```bash
+kubectl create secret generic auth -n stage --from-literal=username=admin --from-literal=password=secretPassword
+```
+
+Define a ClusterAddonsConfiguration:
+```yaml
+apiVersion: addons.kyma-project.io/v1alpha1
+kind: ClusterAddonsConfiguration
+metadata:
+  name: addons-cfg-sample
+spec:
+  repositories:
+    # HTTPS protocol with basic authorization provided.
+    - url: "https://{username}:{password}@repository.addons.com/index.yaml"
+      secretRef:
+        name: auth
+        namespace: stage     
+```
+
+  </details>
+  <details>
+  <summary>
+  Git SSH
+  </summary>
+  
+  The Git SSH protocol requires an sshkey argument which must be base64 encoded SSH private key. You can find more about generating key pair for Github in the [document](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key).
+  Once you created the key pair (*id_rsa* and *id_rsa.pub*) encode the private one:
```

Comment:
```suggestion
  Once you created the key pair encode the private one (*id_rsa*):
```
maybe sth like that? If you have (*id_rsa* and *id_rsa.pub*)  it's quite strange because you are referring to both of them but then use only one.